sources/platform/server/auth-source-control.md

```diff
@@ -182,6 +183,28 @@ If you came to this page to configure Gitlab while installing Shippable server f
 
 If you already have a working Shippable Server instance and just added Gitlab auth, you should be able to sign in with your Gitlab credentials at this time.  
 
+## Gerrit
+
+The following steps configure Gerrit as an authorization provider for your Shippable Server instance:
+
+* Select **GERRIT**.
+* Enter SCM **name** for Gerrit. SCM name is an optional field.
+* Enter your Gerrit **URL**.
+
+### Configure Shippable to trigger builds automatically
```

Comment:
I don't think this should be a separate section. It's a continuation of the Gerrit configuration from above. We should explain it with something like this:

* To automatically trigger builds on Gerrit events, Shippable needs SSH access to your Gerrit server with privileges to run the `gerrit stream-events` command. Create a user on Gerrit with this privilege and enter the user name in Admiral.
* Enter your Gerrit SCM instance's **SSH port**. The default is usually 29418.
* Copy `Public Key` from admiral UI and add this to the Gerrit user you created above.